src/akkd/Header/Header.module.css

```diff
@@ -0,0 +1,40 @@
+@value gutter, md from 'theme.module.css';
+
+.header {
+	color: var(--theme-text);
+	position: relative;
+	margin: calc(2rem - .14em) 0 1rem;

```

Comment:
nit; stuff like the `.14em` would work well as a `@value`

---

purely OOI, where's the .14 coming from? existing semantic stuff?

Reply:
Yeah, that's exactly where it's coming from (but rounded, because before it was something like .14328582958960368)
I can absolutely change it to something more static and/or rounded, but wasn't sure what kind of value would work best.

Reply:
Huh, so it is. Wonder how they're getting that value themselves - 10/10 bet it's some pre-processor calc.